driver-core/src/main/com/mongodb/client/model/changestream/OperationType.java

```diff
@@ -55,6 +55,20 @@
      */
     DROP("drop"),
 
+    /**
+     * The dropDatabase operation type
+     *
+     * @since 3.8.2
+     */
+    DROP_DATABASE("dropDatabase"),
+
+    /**
+     * The rename operation type
+     *
+     * @since 3.8.2
+     */
+    RENAME("rename"),
```

Comment:
@rozza I wasn't sure if this should be "rename" or "renameCollection". It seems like according to the [documentation](https://docs.mongodb.com/manual/reference/command/renameCollection/#dbcmd.renameCollection) it's "renameCollection". Should we stay consistent with that?

Reply:
RENAME is consistent with DROP.  Our convention here seems to be to name based on the string from the server.  Certainly fix up the Javadoc to say that it's about renaming a collection.

Reply:
Done. Added some small clarification to the Javadoc